simple/templates/scale_object.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if .Values.KEDA }}
+  {{- if (and .Values.KEDA.ScaledObject (or .Values.hpa .Values.hpav2)) }}
+    {{ fail "Don't create ScaledObject and HPA at same time." }}
+  {{- end }}
+  {{- if .Values.KEDA.ScaledObject }}
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Release.Namespace }}
+spec: {{ toYaml .Values.KEDA.ScaledObject.spec | nindent 2 }}
+  scaleTargetRef:
+    name: {{ .Values.name }}
```

Comment:
sorry, would like to know if this should be
```
scaleTargetRef:
    deploymentName: {deployment-name} # must be in the same namespace as the ScaledObject
    containerName: {container-name}  #Optional
```

Reply:
I was wrong, new version 2.5 is :
```
scaleTargetRef:
    apiVersion:    {api-version-of-target-resource}  # Optional. Default: apps/v1
    kind:          {kind-of-target-resource}         # Optional. Default: Deployment
    name:          {name-of-target-resource}
```
thanks! 👍

Reply:
I was wrong, new version 2.5 is:
```
scaleTargetRef:
    apiVersion:    {api-version-of-target-resource}  # Optional. Default: apps/v1
    kind:          {kind-of-target-resource}         # Optional. Default: Deployment
    name:          {name-of-target-resource}
```
thanks!